src/components/SchedulePage/Schedule.js

```diff
@@ -36,7 +40,11 @@ const SchedulePage = () => {
 		<Typography align='left' component='h1' variant='h4' className={classes.title}>
 			Schedule
 		</Typography>
-		<ComingSoon alignment='left'/>
+		<Typography align='left' component='h4' className={classes.note}>
```

Comment:
```suggestion
		<Typography align='left' variant='h4' className={classes.note}>
```

The component prop builds this Typography component as an 'h4' HTML tag eventually in the DOM. Semantically this is not a heading, but a subtitle. However, if you like the styling of 'h4' you can use the `variant` prop which will style it like an `h4` tag without actually using an `h4` tag in the final HTML. You might also want to experiment with some of their other variants like `subtitle` for more styling. You can read about it in the [Typography API](https://mui.com/api/typography/) in the docs.